cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                                 icon_url=author.avatar_url)
+                await message.edit(embed=embed)
+
+                return
+
+            if reaction_emoji == "⏹":
+                await leave(message)
+                return
+
+        async def listcustomreacts(message, proposals):
+            if proposals:
+                current_list = self.proposal_list
+            else:
+                current_list = self.reaction_list
+
+            if not current_list:
+                if proposals:
+                    title = 'There are currently no custom reaction proposals in this server'
+                else:
+                    title = 'There are currently no custom reactions in this server'
+                embed = discord.Embed(title=title, delete_after=60)
+                await message.edit(embed=embed)
+                return
+
+            index_list = [
+                '[{}]'.format(i + 1) for i in range(len(current_list))
+            ]
+
+            prompts_and_responses = [
+                'Prompt: {}\nResponse: {}'.format(reaction[1], reaction[2])
+                for reaction in current_list
+            ]
+
+            reaction_dict = {
+                "names": index_list,
+                "values": prompts_and_responses
+            }
+
+            embed = discord.Embed(title="Loading...")
+            await message.edit(embed=embed)
+
+            await message.add_reaction("⏪")
+            await message.add_reaction("◀")
+            await message.add_reaction("▶")
+            await message.add_reaction("⏩")
+            await message.add_reaction("⏹")
+            await message.add_reaction("🆗")
+
+            if proposals:
+                title = "Current custom reaction proposals\n" \
+                        "Click on 🆗 to approve, reject, edit, or see more information on one of them"
+            else:
+                title = "Current custom reactions\nClick on 🆗 to edit or see more information on one of them"
+
+            p = Pages(ctx,
+                      msg=message,
+                      item_list=reaction_dict,
+                      title=title,
+                      display_option=(2, 10),
+                      editable_content_emoji="🆗",
+                      return_user_on_edit=True)
+
+            user_modifying = await p.paginate()
+            while p.edit_mode:
+                await message.clear_reactions()
+                if proposals:
+                    title = "Current custom reaction proposals\n" \
+                            "{}: Write the number of the custom reaction proposal you want to " \
+                            "approve, reject, edit, or see more information on".format(user_modifying)
+                else:
+                    title = "Current custom reactions\n" \
+                            "{}: Write the number of the custom reaction you want to edit or see more " \
+                            "information on".format(user_modifying)
```

Comment:
use variables here too – see my comment with a reply to myself below

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764